pkg/logql/engine_test.go

```diff
@@ -2410,6 +2410,36 @@ func TestEngine_MaxSeries(t *testing.T) {
 	}
 }
 
+func TestEngine_MaxRangeInterval(t *testing.T) {
+	eng := NewEngine(EngineOpts{}, getLocalQuerier(100000), &fakeLimits{rangeLimit: 24 * time.Hour}, log.NewNopLogger())
```

Comment:
for the second test case we need to adjust `maxSeries` limit to not receive an error that we reached the maximum of series ;) 
```suggestion
	eng := NewEngine(EngineOpts{}, getLocalQuerier(100000), &fakeLimits{rangeLimit: 24 * time.Hour, maxSeries: 100000}, log.NewNopLogger())
```